docs/resources/samples/convert-csv.md

```diff
@@ -0,0 +1,86 @@
+---
+title: 'Convert CSV files to Excel workbooks'
+description: 'Learn how to use Office Scripts and Power Automate to create .xlsx files from .csv files.'
+ms.date: 07/19/2021
+localization_priority: Normal
+---
+
+# Convert CSV files to Excel workbooks
+
+Many services export data as comma-separated value (CSV) files. This solution automates the process of converting those CSV files to Excel workbooks in the .xlsx file format. It uses a [Power Automate](https://flow.microsoft.com) flow to find .csv files in a OneDrive folder and an Office Script to copy the data from the .csv file into a new Excel workbook.
+
+## Solution
+
+1. Store the .csv files and a blank "Template" .xlsx file in a OneDrive folder.
+1. Create an Office Script to parse the CSV data into a range.
+1. Create a Power Automate flow to read the .csv files and pass their contents to the script.
+
+## Sample files
+
+Download <a href="https://github.com/OfficeDev/office-scripts-docs/blob/master/docs/resources/samples/convert-csv-example.zip?raw=true">convert-csv-example.zip</a> to get the template .xlsx file and two sample .csv files. Extract the files into a folder in your OneDrive. This sample assumes the folder is named "output".
+
+## Sample code: Insert comma-separated values into a workbook
+
+```TypeScript
+function main(workbook: ExcelScript.Workbook, csv: string) {
+  /* Convert the csv data into a 2D array. */
+  // Trim the trailing new line.
+  csv = csv.trim();
+
+  // Split each line into a row.
+  let rows = csv.split("\r\n");
+  let data : string[][] = [];
+  rows.forEach((value) => {
+    /*
+     * For each row, match the comma-separated sections.
+     * For more information on how to use regular expressions to parse CSV files,
+     * see this Stack Overflow post: https://stackoverflow.com/a/48806378/9227753
+     */
+    data.push(value.match(/(?:,|\n|^)("(?:(?:"")*[^"]*)*"|[^",\n]*|(?:\n|$))/g));
+  });
+
+  // Put the data in the worksheet.
+  let sheet = workbook.getWorksheet("Sheet1");
+  let range = sheet.getRangeByIndexes(0, 0, data.length, data[0].length);
+  range.setValues(data);
+
+  // Add any formatting or table creation that you want.
+}
+```
+
+## Power Automate flow: Create new .xlsx files
+
+1. Sign into [Power Automate](https://flow.microsoft.com) and create a new **Scheduled cloud flow**.
+1. Set the flow to **Repeat every** "1" "Day" and select **Create**.
+1. Get the template Excel file. This is the basis for all the converted .csv files. Add a **New step** that uses the **OneDrive for Business** connector and the **Get file content** action. Provide the file path to the "Template.xlsx" file.
+    1. **File**: /output/Template.xlsx
+1. Rename the **Get file content** step by going to the **...** menu and selecting the **Rename** option. Change the step name to "Get Excel template".
```

Comment:
Dialog might not be the right term, but **...** menu confused me, so I suggest adding a description of the menu location. 
```suggestion
1. Rename the **Get file content** step by going to the **...** menu in the upper right corner of the dialog and selecting the **Rename** option. Change the step name to "Get Excel template".
```

Reply:
Good call. How does this sound?

Rename the **Get file content** step by going the **...** menu of that step (in the upper right corner of the connector) and selecting the **Rename** option. Change the step name to "Get Excel template".

Reply:
Sounds good to me!